esmvaltool/recipes/ref/recipe_ref_sea_ice_area_basic.yml

```diff
@@ -0,0 +1,168 @@
+# ESMValTool
+---
+documentation:
+  title: Seasonal cycle of NH/SH sea ice area
+  description: >
+    Example plot of the seasonal cycle of NH and SH sea ice area (here: CMIP6 model MPI-ESM1-2-LR).
+  authors:
+    - lauer_axel
+  maintainer:
+    - lauer_axel
+
+
+datasets:
+  - {dataset: MPI-ESM1-2-LR, grid: gn, benchmark_dataset: true}
+
+preprocessors:
+
+  pp_nh:
+    extract_region:
+      start_longitude: 0
+      end_longitude: 360
+      start_latitude: 30
+      end_latitude: 90
+    climate_statistics:
+      period: month
+    area_statistics:
+      operator: sum
+    convert_units:
+      units: 1e6 km2
+
+  pp_nh_sep:
+    extract_region:
+      start_longitude: 0
+      end_longitude: 360
+      start_latitude: 30
+      end_latitude: 90
+    extract_month:
+      month: 9
+    area_statistics:
+      operator: sum
+    convert_units:
+      units: 1e6 km2
+
+  pp_sh:
+    extract_region:
+      start_longitude: 0
+      end_longitude: 360
+      start_latitude: -90
+      end_latitude: -30
+    climate_statistics:
+      period: month
+    area_statistics:
+      operator: sum
+    convert_units:
+      units: 1e6 km2
+
+  pp_sh_feb:
+    extract_region:
+      start_longitude: 0
+      end_longitude: 360
+      start_latitude: -90
+      end_latitude: -30
+    extract_month:
+      month: 2
+    area_statistics:
+      operator: sum
+    convert_units:
+      units: 1e6 km2
+
+
+diagnostics:
+
+  siarea_seas:
+    description: Create seasonal cycle plot of sea ice area.
+    variables:
+      sea_ice_area_nh: &var_basic
+        short_name: siconc
+        timerange: '1995/2014'
+        preprocessor: pp_nh
+        project: CMIP6
+        mip: SImon
+        exp: historical
+        ensemble: r1i1p1f1
+        grid: gr
+        reference_dataset: OSI-450-nh
+        additional_datasets:
+        - {dataset: OSI-450-nh, project: OBS, type: reanaly, version: v3, mip: OImon, tier: 2,
+           supplementary_variables: [{short_name: areacello, mip: fx}]}
+#        - {dataset: ESACCI-SEAICE, project: OBS6, tier: 2, type: sat, version: L4-SICONC-RE-SSMI-12.5kmEASE2-fv3.0-SH,
+#           supplementary_variables: [{short_name: areacello, mip: Ofx}]}
+      sea_ice_area_sh:
+        <<: *var_basic
+        preprocessor: pp_sh
+        reference_dataset: OSI-450-sh
+        additional_datasets:
+        - {dataset: OSI-450-sh, project: OBS, type: reanaly, version: v3, mip: OImon, tier: 2,
+           supplementary_variables: [{short_name: areacello, mip: fx}]}
+    scripts:
+      allplots:
+        script: monitor/multi_datasets.py
+        plot_folder: '{plot_dir}'
+        group_variables_by: variable_group
+        facet_used_for_labels: dataset
+        plots:
+          annual_cycle: &plot_basic
+            annual_mean_kwargs: False
+            pyplot_kwargs:
+              title: 'Sea ice area (million km2)'
+            plot_kwargs:
+              'MPI-ESM1-2-LR':
```

Comment:
Do we want this to be hard-coded? It probably would need to be a little more flexible when implemented in the REF so that it just refers to the model that is used.

Reply:
Good point. Changed here: https://github.com/ESMValGroup/ESMValTool/pull/3891/commits/87b2ad56a5c9812c2550abae22bbb3972e310d09